### ISSUE TITLE
[SCRAPER] - madewithlau.com

### ISSUE THREAD

**Issue by kdliu86** [mealie-recipes/mealie#3229]
### First Check

- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have verified that this issue _is not_ related to the underlying library
[hhyrsev/recipe-scrapers](https://github.com/hhursev/recipe-scrapers) by **1)** checking
the [debugger](https://demo.mealie.io/g/home/r/create/debug) and data is returned, **2)**
verifying that there _are_ errors in the log related to application level code, or
**3)** verified that the site provides recipe data, or is otherwise supported by
[hhyrsev/recipe-scrapers](https://github.com/hhursev/recipe-scrapers)

- [X] This issue can be replicated on the demo site (https://demo.mealie.io/)

### Please provide 1-5 example URLs that are having errors

https://www.seriouseats.com/serious-eats-halal-cart-style-chicken-and-rice-white-sauce-recipe

I'm unable to scrape serious eats on my local docker container, but it works fine in the demo

### Please provide your logs for the Mealie container `docker logs <container-id> > mealie.logs`

`failed to get console mode for stdout: The handle is invalid.
Switching to dedicated user

        User uid:    1000
        User gid:    1000
        
INFO: 27-Feb-24 11:06:38 	Database connection established.
INFO: 27-Feb-24 11:06:38 	Context impl SQLiteImpl.
INFO: 27-Feb-24 11:06:38 	Will assume non-transactional DDL.
INFO: 27-Feb-24 11:06:38 	Checking for migration data fixes
WARNING: 27-Feb-24 11:06:38 	"get_all" method is deprecated; use "page_all" instead
INFO: 27-Feb-24 11:06:42 	-----SYSTEM STARTUP----- 

INFO: 27-Feb-24 11:06:42 	------APP SETTINGS------
INFO: 27-Feb-24 11:06:42 	{
    "theme": {
        "light_primary": "#E58325",
        "light_accent": "#007A99",
        "light_secondary": "#973542",
        "light_success": "#43A047",
        "light_info": "#1976D2",
        "light_warning": "#FF6D00",
        "light_error": "#EF5350",
        "dark_primary": "#E58325",
        "dark_accent": "#007A99",
        "dark_secondary": "#973542",
        "dark_success": "#43A047",
        "dark_info": "#1976D2",
        "dark_warning": "#FF6D00",
        "dark_error": "#EF5350"
    },
    "PRODUCTION": true,
    "BASE_URL": "http://localhost:8083",
    "STATIC_FILES": "/spa/static",
    "IS_DEMO": false,
    "API_PORT": 9000,
    "API_DOCS": true,
    "TOKEN_TIME": 48,
    "LOG_LEVEL": "INFO",
    "GIT_COMMIT_HASH": "42a33cd993b4638f181d4083a7c5d21bf110e4e4",
    "ALLOW_SIGNUP": true,
    "SECURITY_MAX_LOGIN_ATTEMPTS": 5,
    "SECURITY_USER_LOCKOUT_TIME": 24,
    "DB_ENGINE": "sqlite",
    "DEFAULT_GROUP": "Home",
    "SMTP_HOST": null,
    "SMTP_PORT": "587",
    "SMTP_FROM_NAME": "Mealie",
    "SMTP_FROM_EMAIL": null,
    "SMTP_AUTH_STRATEGY": "TLS",
    "LDAP_AUTH_ENABLED": false,
    "LDAP_SERVER_URL": null,
    "LDAP_TLS_INSECURE": false,
    "LDAP_TLS_CACERTFILE": null,
    "LDAP_ENABLE_STARTTLS": false,
    "LDAP_BASE_DN": null,
    "LDAP_QUERY_BIND": null,
    "LDAP_QUERY_PASSWORD": null,
    "LDAP_USER_FILTER": null,
    "LDAP_ADMIN_FILTER": null,
    "LDAP_ID_ATTRIBUTE": "uid",
    "LDAP_MAIL_ATTRIBUTE": "mail",
    "LDAP_NAME_ATTRIBUTE": "name",
    "TESTING": false
}
INFO:     172.22.0.1:34258 - "GET /admin/site-settings HTTP/1.1" 307 Temporary Redirect
INFO:     172.22.0.1:34258 - "GET /admin/site-settings/ HTTP/1.1" 200 OK
INFO:     172.22.0.1:34258 - "GET /_nuxt/ef8a64b.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:34268 - "GET /_nuxt/0472ea9.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:34280 - "GET /_nuxt/8254a36.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:34268 - "GET /_nuxt/d94ba00.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:34268 - "GET /api/users/self HTTP/1.1" 200 OK
INFO:     172.22.0.1:34268 - "GET /api/media/users/b26902ca-e857-4874-a49c-446a11846e8e/profile.webp?cacheKey=1234 HTTP/1.1" 200 OK
INFO:     172.22.0.1:34258 - "GET /api/admin/about/check HTTP/1.1" 200 OK
INFO:     172.22.0.1:34280 - "GET /api/admin/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:35626 - "GET /_nuxt/4e32b0d.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35642 - "GET /_nuxt/1df36d4.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35642 - "GET /api/users?page=1&perPage=-1 HTTP/1.1" 200 OK
INFO:     172.22.0.1:35642 - "GET /_nuxt/9671346.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35666 - "GET /_nuxt/ff34c85.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35680 - "GET /_nuxt/820b022.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35626 - "GET /api/groups/statistics HTTP/1.1" 200 OK
INFO:     172.22.0.1:35642 - "GET /api/groups/cookbooks?page=1&perPage=-1&orderBy=position&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:35654 - "GET /api/users/self/group HTTP/1.1" 200 OK
INFO:     172.22.0.1:35654 - "GET /_nuxt/eebbea5.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35642 - "GET /_nuxt/70f1f3c.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35626 - "GET /_nuxt/3f8900d.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35680 - "GET /_nuxt/ef7b985.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35626 - "GET /_nuxt/img/manage-group-settings.9e6a4a2.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:35680 - "GET /_nuxt/img/manage-cookbooks.4dc1c10.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:35626 - "GET /_nuxt/09dc712.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35666 - "GET /_nuxt/a89aa77.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35654 - "GET /_nuxt/5b8a0d2.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35642 - "GET /_nuxt/492b20f.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35680 - "GET /_nuxt/89e3151.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35680 - "GET /api/organizers/categories?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:35654 - "GET /api/organizers/tags?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:35642 - "GET /api/foods?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:35626 - "GET /api/organizers/tools?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:35666 - "GET /api/recipes?page=1&perPage=64&orderBy=created_at&orderDirection=desc&paginationSeed=1709064420778&searchSeed=1709064420778&search=&requireAllCategories=false&requireAllTags=false&requireAllTools=false&requireAllFoods=false HTTP/1.1" 200 OK
INFO:     172.22.0.1:35666 - "GET /api/media/recipes/313d206f-a588-47ed-8ac1-b51d389c259c/images/min-original.webp?rnd=1&version=k3AP HTTP/1.1" 200 OK
INFO:     127.0.0.1:32938 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:35072 - "GET /api/recipes?page=1&perPage=64&orderBy=created_at&orderDirection=desc&paginationSeed=1709064440056&searchSeed=1709064440056&search=https%3A%2F%2Fwww.madewithlau.com%2Frecipes%2Fsiu-yuk-crispy-pork-belly&requireAllCategories=false&requireAllTags=false&requireAllTools=false&requireAllFoods=false HTTP/1.1" 200 OK
INFO:     172.22.0.1:35074 - "GET /_nuxt/a831122.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:35072 - "GET /_nuxt/a46644d.js HTTP/1.1" 200 OK
INFO: 27-Feb-24 11:07:25 	HTTP Request: GET https://www.madewithlau.com/recipes/siu-yuk-crispy-pork-belly "HTTP/1.1 200 OK"
INFO: 27-Feb-24 11:07:25 	HTTP Request: GET https://www.madewithlau.com/recipes/siu-yuk-crispy-pork-belly "HTTP/1.1 200 OK"
INFO:     172.22.0.1:35074 - "POST /api/recipes/create-url HTTP/1.1" 400 Bad Request
INFO:     127.0.0.1:33854 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:46296 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:34720 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:46648 - "GET /api/foods?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:46662 - "GET /api/organizers/tools?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:46634 - "GET /api/organizers/categories?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:46618 - "GET /api/recipes?page=1&perPage=64&orderBy=created_at&orderDirection=desc&paginationSeed=1709064532523&searchSeed=1709064532523&search=&requireAllCategories=false&requireAllTags=false&requireAllTools=false&requireAllFoods=false HTTP/1.1" 200 OK
INFO:     127.0.0.1:56952 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:45194 - "GET /api/recipes?page=1&perPage=64&orderBy=created_at&orderDirection=desc&paginationSeed=1709064546630&searchSeed=1709064546630&search=https%3A%2F%2Fthewoksoflife.com%2Fcantonese-roast-pork-belly%2F&requireAllCategories=false&requireAllTags=false&requireAllTools=false&requireAllFoods=false HTTP/1.1" 200 OK
INFO: 27-Feb-24 11:09:12 	HTTP Request: GET https://thewoksoflife.com/cantonese-roast-pork-belly/ "HTTP/1.1 200 OK"
INFO: 27-Feb-24 11:09:12 	Image URL: ['https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1.jpg', 'https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1-500x500.jpg', 'https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1-500x375.jpg', 'https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1-480x270.jpg']
INFO: 27-Feb-24 11:09:12 	HTTP Request: HEAD https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1-500x500.jpg "HTTP/1.1 200 OK"
INFO: 27-Feb-24 11:09:12 	HTTP Request: HEAD https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1.jpg "HTTP/1.1 200 OK"
INFO: 27-Feb-24 11:09:12 	HTTP Request: HEAD https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1-500x375.jpg "HTTP/1.1 200 OK"
INFO: 27-Feb-24 11:09:12 	HTTP Request: HEAD https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1-480x270.jpg "HTTP/1.1 200 OK"
INFO: 27-Feb-24 11:09:12 	HTTP Request: GET https://thewoksoflife.com/wp-content/uploads/2015/03/cantonese-roast-pork-belly-13-1.jpg "HTTP/1.1 200 OK"
INFO: 27-Feb-24 11:09:12 	original.jpg minified
INFO: 27-Feb-24 11:09:12 	Tiny image saved
INFO:     172.22.0.1:45210 - "POST /api/recipes/create-url HTTP/1.1" 201 Created
INFO:     172.22.0.1:45210 - "GET /_nuxt/5ba8ded.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:45210 - "GET /_nuxt/556e386.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:45224 - "GET /_nuxt/6f5c0f4.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:45224 - "GET /api/recipes/cantonese-roast-pork-belly HTTP/1.1" 200 OK
INFO:     172.22.0.1:45224 - "GET /api/media/recipes/fa149c1d-560e-486d-94d2-6de469998057/images/original.webp?rnd=1&version=IIPR HTTP/1.1" 200 OK
INFO:     172.22.0.1:45240 - "GET /api/recipes/cantonese-roast-pork-belly/comments HTTP/1.1" 200 OK
INFO:     172.22.0.1:45210 - "GET /api/organizers/tools?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39186 - "GET /api/groups/statistics HTTP/1.1" 200 OK
INFO:     172.22.0.1:39186 - "GET /_nuxt/img/manage-members.12f372e.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /_nuxt/img/manage-webhooks.b22331c.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:39190 - "GET /_nuxt/img/manage-data-migrations.82e18a2.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:39218 - "GET /_nuxt/img/manage-recipes.bd20bc7.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:39204 - "GET /_nuxt/img/manage-notifiers.fe43394.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /api/users/self/group HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /_nuxt/5804e4b.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39186 - "GET /_nuxt/383f132.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39218 - "GET /_nuxt/0832465.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /_nuxt/20999a1.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39190 - "GET /_nuxt/9481ef9.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39204 - "GET /_nuxt/e2148b2.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /_nuxt/a86d438.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39204 - "GET /_nuxt/71028e4.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39190 - "GET /_nuxt/ae58f12.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39190 - "GET /api/organizers/categories?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39204 - "GET /api/foods?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /api/organizers/tools?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/recipes?page=1&perPage=64&orderBy=created_at&orderDirection=desc&paginationSeed=1709064560974&searchSeed=1709064560974&search=&requireAllCategories=false&requireAllTags=false&requireAllTools=false&requireAllFoods=false HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/media/recipes/fa149c1d-560e-486d-94d2-6de469998057/images/min-original.webp?rnd=1&version=IIPR HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /_nuxt/b22299b.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /_nuxt/b05c043.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/organizers/tools?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /api/organizers/tools?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /_nuxt/b617de2.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /_nuxt/f367be5.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /api/organizers/categories?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/organizers/categories?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /_nuxt/b237774.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /_nuxt/478f080.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/groups/self HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/recipes/timeline/events?page=1&perPage=32&orderBy=timestamp&orderDirection=asc&queryFilter=recipe.group_id%3D%2287044c89-b8bf-49c6-9131-6ef5cd2f55d8%22 HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /api/recipes/fa149c1d-560e-486d-94d2-6de469998057 HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/recipes/313d206f-a588-47ed-8ac1-b51d389c259c HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /_nuxt/a8c3eea.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/groups/shopping/lists?page=1&perPage=-1 HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /_nuxt/img/shopping-cart.1ff7637.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /_nuxt/a3313cf.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:53032 - "GET /_nuxt/d62cac1.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /_nuxt/41def40.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/groups/mealplans?page=1&perPage=-1&start_date=2024-02-27&end_date=2024-03-04 HTTP/1.1" 200 OK
INFO:     172.22.0.1:53032 - "GET /api/foods?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:53038 - "GET /api/organizers/tools?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39188 - "GET /api/organizers/categories?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /api/recipes?page=1&perPage=64&orderBy=created_at&orderDirection=desc&paginationSeed=1709064567523&searchSeed=1709064567523&search=&requireAllCategories=false&requireAllTags=false&requireAllTools=false&requireAllFoods=false HTTP/1.1" 200 OK
INFO:     172.22.0.1:53038 - "GET /api/admin/about/check HTTP/1.1" 200 OK
INFO:     172.22.0.1:39232 - "GET /api/admin/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:43696 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:60668 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:58336 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:50694 - "GET /g/home/r/create/url?recipe_import_url=https://www.seriouseats.com/perfect-roast-chicken-recipe-8384377?print&use_keywords=1&edit=1 HTTP/1.1" 200 OK
INFO:     172.22.0.1:50694 - "GET /_nuxt/ef8a64b.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50704 - "GET /_nuxt/8449c9e.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50708 - "GET /_nuxt/0472ea9.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50714 - "GET /_nuxt/8254a36.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50708 - "GET /_nuxt/9d88ba8.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50704 - "GET /_nuxt/a46644d.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50714 - "GET /_nuxt/a831122.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50714 - "GET /api/app/about/theme HTTP/1.1" 200 OK
INFO:     172.22.0.1:50730 - "GET /_nuxt/manifest.260e8103.json HTTP/1.1" 200 OK
INFO:     172.22.0.1:50714 - "GET /icons/android-chrome-192x192.png HTTP/1.1" 200 OK
INFO:     172.22.0.1:50704 - "GET /sw.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50714 - "GET /_nuxt/fonts/Roboto-400-latin21.b009a76.woff2 HTTP/1.1" 200 OK
INFO:     172.22.0.1:50704 - "GET /_nuxt/fonts/Roboto-500-latin28.f25d774.woff2 HTTP/1.1" 200 OK
INFO:     172.22.0.1:50708 - "GET /api/explore/cookbooks/home?page=1&perPage=-1&orderBy=position&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:50694 - "GET /icons/icon-x64.png HTTP/1.1" 200 OK
INFO:     172.22.0.1:50744 - "POST /api/recipes/create-url HTTP/1.1" 401 Unauthorized
INFO:     172.22.0.1:50752 - "GET /api/organizers/tags?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 401 Unauthorized
INFO:     172.22.0.1:50708 - "GET /_nuxt/img/recipes-create.ca6e04e.svg HTTP/1.1" 200 OK
INFO:     172.22.0.1:50752 - "GET /_nuxt/79eeeb6.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50752 - "GET /favicon.ico HTTP/1.1" 200 OK
INFO:     172.22.0.1:50752 - "GET /g/home/r/create/url?recipe_import_url=https://www.seriouseats.com/perfect-roast-chicken-recipe-8384377&use_keywords=1&edit=1 HTTP/1.1" 200 OK
INFO:     172.22.0.1:50708 - "GET /api/organizers/tags?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 401 Unauthorized
INFO:     172.22.0.1:50744 - "POST /api/recipes/create-url HTTP/1.1" 401 Unauthorized
INFO:     172.22.0.1:50752 - "GET /api/explore/cookbooks/home?page=1&perPage=-1&orderBy=position&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:50752 - "GET /sw.js HTTP/1.1" 304 Not Modified
INFO:     127.0.0.1:37198 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:33900 - "POST /api/recipes/create-url HTTP/1.1" 401 Unauthorized
INFO:     127.0.0.1:53564 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:32910 - "GET /g/home/r/create/url?recipe_import_url=https://www.seriouseats.com/serious-eats-halal-cart-style-chicken-and-rice-white-sauce-recipe&use_keywords=1&edit=1 HTTP/1.1" 200 OK
INFO:     172.22.0.1:32910 - "GET /api/explore/cookbooks/home?page=1&perPage=-1&orderBy=position&orderDirection=asc HTTP/1.1" 200 OK
INFO:     172.22.0.1:32924 - "POST /api/recipes/create-url HTTP/1.1" 401 Unauthorized
INFO:     172.22.0.1:32932 - "GET /api/organizers/tags?page=1&perPage=-1&orderBy=name&orderDirection=asc HTTP/1.1" 401 Unauthorized
INFO:     172.22.0.1:32932 - "GET /sw.js HTTP/1.1" 304 Not Modified
INFO:     127.0.0.1:44646 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:50084 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:55290 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:50340 - "GET / HTTP/1.1" 200 OK
INFO:     172.22.0.1:50340 - "GET /_nuxt/383a928.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50340 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:50356 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:54918 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     172.22.0.1:50356 - "GET /_nuxt/89e3151.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50340 - "GET /_nuxt/09dc712.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50368 - "GET /_nuxt/5b8a0d2.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50362 - "GET /_nuxt/492b20f.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50378 - "GET /_nuxt/a89aa77.js HTTP/1.1" 200 OK
INFO:     172.22.0.1:50378 - "GET /sw.js HTTP/1.1" 304 Not Modified
INFO:     127.0.0.1:54762 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:40528 - "GET /api/app/about HTTP/1.1" 200 OK
INFO:     127.0.0.1:41468 - "GET /api/app/about HTTP/1.1" 200 OK
`

### Deployment

Docker (Windows)

**Comment by kdliu86** [comment]
Not really sure what's wrong, I tried a bunch of other sites (woksoflife, pailin's kitchen, omnivore cookbook) and they all scrape fine.  Madewithlau and seriouseats won't scrape, throwing 400 and 500 errors.

**Comment by kdliu86** [comment]
Posted in Discord, for whatever reason SeriousEats started parsing.  Madewithlau still doesn't work though.

**Comment by Kuchenpirat** [comment]
Hi, 
thanks for raising this issue. Seems like your seriouseats problems might have been a network etc problem. Glad to hear it has been resolved. 

As for the inability to scraper madewithlau, i can confirm this. 
I tested it with the scraper directly and can neither recive data in normal mode, which should work since this website is supported, nor in wild mode. 

So it seems that it is a upstream issue and you would have to [open an issue](https://github.com/hhursev/recipe-scrapers/issues/new/choose) with them. 

Below the returned scraper data for reference:
```
{
  'author': None, 
  'canonical_url': 'https://www.madewithlau.com/recipes/siu-yuk-crispy-pork-belly', 
  'category': None, 
  'host': 'madewithlau.com', 
  'image': None, 
  'ingredient_groups': [{'ingredients': [], 'purpose': None}], 
  'ingredients': [], 
  'instructions': '', 
  'instructions_list': [], 
  'site_name': None
}
```

Also https://www.madewithlau.com/recipes/siu-yuk-crispy-pork-belly since the url is a bit hidden above.
